docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md

```diff
@@ -218,18 +218,7 @@ Now that you have set up collection for AWS EC2 metrics install the Sumo Logic a
 
 To install the app:
```

Comment:
```suggestion
```

Reply:
"To install the app:" appears twice in the article because it's also the first sentence of the reuse snippet.